java/maven_deps.bzl

```diff
@@ -97,7 +97,7 @@ def selenium_java_deps():
             "org.mockito:mockito-core:4.11.0",
             "org.slf4j:slf4j-api:2.0.7",
             "org.slf4j:slf4j-jdk14:2.0.7",
-            "org.testng:testng:7.7.1",
+            "org.apache.logging.log4j:log4j-core:2.20.0",
```

Comment:
Why do we need log4j? Are we using that transitively?

Reply:
`com.github.spotbugs:spotbugs` does use it and so it is added transitively.

Reply:
Is this required for our linting tests? Why do we have it as a runtime dependency?

Reply:
We should also get rid of spotbugs and use something across the project.

Also, those are all the dependencies, when the packages are created then only the runtime are packed.